autoload/youcompleteme.vim

```diff
@@ -105,7 +105,27 @@ endfunction
 
 function! youcompleteme#DisableCursorMovedAutocommands()
     autocmd! ycmcompletemecursormove CursorMoved *
-    autocmd! ycmcompletemecursormove CursorMovedI *
+endfunction
+
+
+function! youcompleteme#EnableTextChangedAutocommands()
+    augroup ycmcompletemetextchange
+        autocmd!
+        if pyeval( 'vimsupport.VimVersionAtLeast("7.3.867")' )
+          autocmd TextChangedI * call s:OnTextChangedInsertMode()
+        else
+          autocmd CursorMovedI * call s:OnTextChangedInsertMode()
+        endif
+    augroup END
+endfunction
+
+
+function! youcompleteme#DisableTextChangedAutocommands()
```

Comment:
For simplify the Disable function why don't we just use:
```viml
autocmd! ycmcompletemetextchange
```

without the event?